tests/unittests/dataset_tests.py

```
@@ -704,11 +731,47 @@ def test_indexes(self):
         dataset.drop_index("cls.label")
         self.assertNotIn("cls.label", dataset.list_indexes())
 
-        compound_index_name = dataset.create_index([("id", 1), ("field", 1)])
-        self.assertIn(compound_index_name, dataset.list_indexes())
+        dataset.create_index([("field", -1)])
+        indexes = dataset.get_index_information()
+        indexes["field"]["key"] == [("field", -1)]
```

Ruff (0.13.1)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)